docs/learn/learn-extrinsics.md

```diff
@@ -0,0 +1,73 @@
+example, the staking pallet contains all functionalities related to staking such as bonding or
+unbonding funds. The combined information of pallets and calls constitutes an **extrinsic**, i.e. a
+transaction that is executed from outside the chain but that triggers an event on the chain.
+Continuing with the staking example, within the staking pallet a nominator can bond funds and
+nominate some validators. The signature of such extrinsic might lead to an event on the chain such
+as a reward payout to that nominator at the end of an era; this is an event inside the chain.
+
+## Extrinsics (a.k.a. Types of Transactions)
+
+Now that we introduced where the term _extrisic_ comes from, we can explain what extrinsics really are. In Substrate, a _"thing that could exist in the transaction pool"_ — more commonly known as "a transaction" in the non-Substrate world — can be one of 3 distinct types, all of which fall under a broader category called "extrinsics". Extrinsics is just a general term to mean "any information that originates from outside a runtime". These are:
+
+- **Signed transactions:** these must contain the signature of the account sending the inbound request to the runtime. With signed transactions, the account used to submit the request typically pays a transaction fee and must sign it using the account's private key.
+- **Unsigned transactions:** these don't carry any information about who submitted the transaction, since the format of this type of transaction doesn't require a signature. You can define what conditions must be met for such a transaction to be valid.
+- **Inherents:** are a special type of unsigned transaction made by block authors which carry information required to build a block such as timestamps, storage proofs and uncle blocks.
+ 
+Here are some key differences between the different types of extrinsics:
+
+- Contrary to signed transactions, unsigned transaction types require implementing custom validation logic which can consume more resources for checking validity compared to signed transactions.
+- Unsigned transactions have no economic deterrent to prevent spam or replay attacks, so custom logic must account for protecting the network from these types of transactions being misused.
+- Inherents exist to address the need of adding some data to a block, whereas signed or unsigned transactions exist to potentially change the state of the blockchain.
+
+## Importance of Verifying Extrinsics
+
+:::info Walk-through Video Tutorial 
+
+Visit the [dedicated support page](https://support.polkadot.network/support/solutions/articles/65000179161-how-can-i-verify-what-extrinsic-i-m-signing-) and see [this technical explainer](https://youtu.be/bxMs-9fBtFk) tutorial to learn about how to verify extrinsics before signing them. The video will also mention potential attacks that can happen to you while signing for transactions. **Do not sign any transaction if you suspect you are or have been under attack.**
```

Comment:
> **Do not sign any transaction if you suspect you are or have been under attack.**

I'd change that to "Do not sign a transaction if you can't verify what you are signing or you suspect you might be signing a different extrinsic than the one intended"

Reply:
Changed